### PR TITLE
Todo card list ui

### PR DESCRIPTION
**Pull request by DDKatch**
screen shot 2018-11-29 at 4 59 48 pm

### CODE REVIEW

**Comment by DDKatch** [comment]
[I as user want to see all added TODO cards](https://trello.com/c/ZNHW35JE/21-i-as-user-want-to-see-all-added-todo-cards)

**Comment by AleksSenkou** [comment]
@DDKatch can you rebase it with latest master?

**Comment by DDKatch** [comment]
@AleksSenkou rebased with master

**Review by AleksSenkou — COMMENTED** [review]
image

This is what I saw at the first page loading:
`Each child in an array or iterator should have a unique "key" prop` should be fixed

File: client/src/containers/TodoCardListContainer/index.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    this.loadTodoCards()
+  }
+
+  render = () => (
+    <div className="col-sm-3 mx-auto">
+      { this.state.todoCards.map(todoCard => <TodoCard description={todoCard.description} />) }
+    </div>
+  );
+
+  async loadTodoCards() {
```
[the remark this answers is not in the document]
Comment (DDKatch) [reply client/src/containers/TodoCardListContainer/index.js]: @AleksSenkou there's a `fetch` method in js https://developer.mozilla.org/ru/docs/Web/API/Fetch_API/Using_Fetch

[the remark this answers is not in the document]
Comment (AleksSenkou) [reply client/src/containers/TodoCardListContainer/index.js]: @DDKatch ok, `fetchList` 😄

**Comment by DDKatch** [comment]
@AleksSenkou again, I don't have such an error.

File: client/src/containers/TodoCardListContainer/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (AleksSenkou) [reply client/src/containers/TodoCardListContainer/index.js]: Most that I've seen was used with `fetch` instead of `load`. `load` is a rails way I'd say

**Comment by AleksSenkou** [comment]
@DDKatch ideas why you don't have such errors?

**Comment by DDKatch** [comment]
@AleksSenkou maybe because of I use Chrome Version 70.0.3538.102

**Comment by DDKatch** [comment]
@AleksSenkou I've fixed
screen shot 2018-11-30 at 5 28 29 pm

How to: https://stackoverflow.com/questions/50000703/google-chrome-does-not-show-errors-in-console

**Comment by DDKatch** [comment]
@datarockets/cuties fixed reviews

File: client/src/components/TodoCardList/index.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import TodoCard from 'src/components/TodoCard'
+
+const TodoCardList = props => (
+  <div className="col-sm-3 mx-auto">
+    { props.todoCards.map(
+      todoCard => <TodoCard key={todoCard.id} description={todoCard.description} />,
```
Comment (AleksSenkou) [inline client/src/components/TodoCardList/index.js]: @DDKatch can we leave `todoCard ` on the previous line? Just a comunity tip

**Review by AleksSenkou — APPROVED** [review]
LGTM with suggestion

**Pull request merged**